nixos/doc/manual/default.nix

```diff
@@ -139,7 +139,7 @@ let
 
   manual-combined = runCommand "nixos-manual-combined"
     { inherit sources;
-      buildInputs = [ libxml2 libxslt ];
+      nativeBuildInputs = [ buildPackages.libxml2 buildPackages.libxslt ];
```

Comment:
Why does this need both `nativeBuildInputs` and `buildPackages`? Isn`t
````
      nativeBuildInputs = [ libxml2 libxslt ];
````

enough?

Reply:
@dezgeg No, because the magic autoresolution for nativeBuildInputs essentially relies on being inside of a `callPackage`d function (this is an oversimplification, but I don't think it's misleading in this context. @Ericson2314 ?). Because we simply have a top-level `with pkgs` in this file, we don't get that benefit.

Note that we are working toward infrastructure that doesn't rely on the "splicing" hack that causes this behavior.

Reply:
Ugh, I see.

Reply:
Yeah that's all correct.